<?xml version="1.0" ?>
<launch>

    <!-- GENERAL ARGUMENTS -->
    <arg name="simulation" default="false"/>
    <arg name="robot_name" default="ur10e"/>

    <!-- ROBOT SETUP -->
    <include file="$(find robot_arm_tools)/launch/Setup_$(arg robot_name).launch">
        <arg name="tool_name" value="pencil"/>
        <arg name="simulation" value="$(arg simulation)"/>
        <arg name="twist_control" value="true"/>
    </include> 

    <!-- MAIN NODE -->
    <node pkg="panda_draws_you" type="panda_draws_you_plane_calibration_node" name="panda_draws_you_plane_calibration_node" output="screen" required="true"/> 
    
    <!-- Run Rviz -->
    <param name="environmentFileName" value="$(find robot_arm_tools)/config/environments/OptiTrack.yaml"/>
    <node pkg="robot_arm_tools" type="robot_arm_tools_environment_node" name="robot_arm_tools_environment_node" output="screen"/>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find robot_arm_tools)/config/DefaultSetup.rviz"/>  <!-- Customisable RViz configuration -->

</launch>